### PR TITLE
Updated <title> translation with Spanish character encoding

### PR DESCRIPTION
**Pull request by mostlybadfly**
This is to update character encoding for the title translation, just needs to be checked to make sure there aren't any literal accented characters.

### CODE REVIEW

File: elements/title.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
   <div class="how-to-remember">
     <div class="how-to-remember-box how-to-remember-spanish">
       <h4>&iquest;C&oacute;mo recordar?</h4>
-      <p>Aunque no se puede juzgar un libro por su portada, se puede hacer una idea buena de qué se trata por el título.  Un título (title en inglés) es una manera rápida de identificar de qué se trata el documento.</p>
+      <p>Aunque no se puede juzgar un libro por su portada, se puede hacer una idea buena de qu&eacute; se trata por el t&iacute;tulo.  Un ti&iacute;tulo (title en ingl&eacute;s) es una manera r&aacute;pida de identificar de qu&eacute; se trata el documento.</p>
```
Comment (JenEEsquivel) [inline elements/title.html]: Thank you :+1:

Comment (ericanaughton) [inline elements/title.html]: @mostlybadfly 
Line 30, 2nd sentence: 
ti&iacute;tulo should be 
t&iacute;tulo 
(an extra i is there)

Comment (mostlybadfly) [inline elements/title.html]: Updated now @ericanaughton :)

**Comment by ericanaughton** [comment]
:+1:

**Comment by mostlybadfly** [comment]
woohoo! so nice to see all these merge :raised_hands:

**Pull request merged**